README.rst

```diff
@@ -128,13 +128,24 @@ in Black documentation, or the article
 How?
 ====
 
-To install, use::
+To install or upgrade, use::
 
-  pip install darker
+  pip install --upgrade darker~=1.6.0
 
 Or, if you're using Conda_ for package management::
 
-  conda install -c conda-forge darker isort
+  conda install -c conda-forge darker~=1.6.0 isort
+  conda update -c conda-forge darker
+
+..
+
+    **Note:** We recommend to use the ``~=`` "`compatible release`_" version specifier
+    to ensure you have the newest Darker before the next compatibility-breaking major
+    release. If a Black update introduces an incompatibility with Darker, we'll issue
+    one Darker point release which prevents upgrading Black, and later a second one
+    which fixes the incompatibility.
```

Comment:
Maybe clarity can be improved here?

Reply:
What happens if the user has updated Black to a incompatible version? Will the new patch update of Darker downgrade Black?

Reply:
Yes, that's what should happen. I should probably clarify that.

Reply:
Although as long as the incompatibliity hasn't made in into a Black release, the patch update of Darker will simply prevent Black from upgrading into that release in the future.

Reply:
I'd hope that by the time Black releases the incompatibility, we'll already have the second Darker patch update published and be compatible.

Reply:
Yes, extending the README would indeed be an improvement.

Reply:
README has now been improved.